### PR TITLE
Add the Renaissance paper.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

File: softdev.bib [hunk 1]
```diff
@@ -2479,3 +2479,25 @@ @inproceedings{rohou15branch
     month = Feb,
     doi= "10.1109/CGO.2015.7054191"
 }
+
+@inproceedings{prokopec19renaissance,
```
Comment (ltratt) [inline softdev.bib]: Can we use the accents directly rather than the old TeX stuff. All our papers expect UTF-8 input these days.

Comment (vext01) [reply softdev.bib]: Yep!

**Comment by vext01** [comment]
Try this.

**Comment by ltratt** [comment]
Please squash.

**Comment by vext01** [comment]
splat

**Pull request merged**